### PR TITLE
docs: Update Russian docs

### PR DESCRIPTION
**Pull request by cogor**
#256

- [x] Update configuration.md
- [x] Update index.md

### CODE REVIEW

File: docs/content/ru/index.md [hunk 1]
```diff
@@ -11,7 +11,8 @@ features:
   - Мощный API конструктора запросов (MongoDB подобный)
   - Подсветка синтаксиса для блоков кода в файлах markdown с помощью PrismJS.
   - Генерация оглавления
-  - работает с Markdown, CSV, YAML, JSON(5), XML
+  - Работает с Markdown, CSV, YAML, JSON(5), XML,
```
Comment (benjamincanac) [inline docs/content/ru/index.md:14]: Is the comma at the end normal?

Comment (cogor) [reply docs/content/ru/index.md:14]: No, it's typo, sorry

Comment (cogor) [reply docs/content/ru/index.md:14]: I will fix it within an hour

**Comment by benjamincanac** [comment]
Thanks @cogor!

**Pull request merged**